docs/Contributors_Guide/add_use_case.rst

```diff
@@ -22,32 +38,397 @@ New use cases will be put in the repository under parm/use_cases/model_applicati
+
+.. note::
+    The following variables do not need to be changed unless DTC moves to a
+    new web server, but they still need to be set to run the instructions
+
+If you are using **bash**::
+
+    # web server that hosts the input data
+    export DTC_WEB_SERVER=mohawk.rap.ucar.edu
+
+    # directory on DTC web server to put new data
+    export DATA_STAGING_DIR=/d2/projects/METplus/METplus_Data_Staging
+
+    # directory on DTC web server that contains the data tarfiles
+    export DATA_TARFILE_DIR=/d2/www/dtcenter/dfiles/code/METplus/METplus_Data
+
+If you are using **csh**::
+
+    # web server that hosts the input data
+    setenv DTC_WEB_SERVER mohawk.rap.ucar.edu
+
+    # directory on DTC web server to put new data
+    setenv DATA_STAGING_DIR /d2/projects/METplus/METplus_Data_Staging
+
+    # directory on DTC web server that contains the data tarfiles
+    setenv DATA_TARFILE_DIR /d2/www/dtcenter/dfiles/code/METplus/METplus_Data
+
+* Run all of the environment variable commands in your shell and verify that
+  they were set correctly.
+
+* Put new dataset into a directory that matches the use case directories, i.e.
+  model_applications/<category> or met_tool_wrapper (formerly
+  met_test) where <category> is the use case category from the list above.
+
+* Set directory paths in the use case config file relative to INPUT_BASE
+  i.e {INPUT_BASE}/model_applications/<category> and set {INPUT_BASE} to your
+  local directory to test
+
+* Create a tarfile on your development machine with the new dataset. Make sure
+  the tarball contains directories model_applications/<category> or
+  met_tool_wrapper (formerly met_test)::
+
+    tar czf ${NEW_DATA_TARFILE} model_applications/${USE_CASE_CATEGORY}
+
+* If you have access to the internal DTC web server,
+  copy over the tarfile to
+  /d2/projects/METplus/METplus_Data_Staging::
+
+    scp ${NEW_DATA_TARFILE} ${DTC_WEB_SERVER}:${DATA_STAGING_DIR}/
+
+* If you do not, upload the tarfile to the RAL FTP::
+
+    ftp -p ftp.rap.ucar.edu
+
+For an example on how to upload data to the ftp site see
+“How to Send Us Data” on the
+`MET Help Webpage <https://dtcenter.org/community-code/model-evaluation-tools-met/met-help-desk>`_.
+
+Adding new data to full sample data tarfile
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Switch to the met_test user on the DTC web server::
+
+    runas met_test
+
+* **Run all of the environment variable commands in your shell (from the first
+  step) and verify that they were set correctly**
+
+* As the met_test user, create a new directory in the METplus_Data web
+  directory named after the branch containing the changes for the new use case.
+  On the DTC web server::
+
+    cd ${DATA_TARFILE_DIR}
+    mkdir ${METPLUS_FEATURE_BRANCH}
+    cd ${METPLUS_FEATURE_BRANCH}
+
+Check if the category tarfile exists already
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Check the symbolic link in the develop directory to determine latest tarball
```

Comment:
The colons are used to signify the start of a literal block. There is no literal block that follows this line, so there is no need for the colons and it may even cause an error.